src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesGcm.macOS.cs

```diff
@@ -11,7 +11,7 @@ public sealed partial class AesGcm
     {
         private FixedMemoryKeyBox _keyBox;
 
-        // CryptoKit added AES.GCM in macOS 10.15, which is our minimum target for macOS.
+        // CryptoKit added AES.GCM in macOS 10.15, which is higher than our minimum target for macOS.
```

Comment:
nit: Shouldn't this be "lower than our minimum target.."?

Reply:
Ah right, I meant to say that our minimum target is higher than 10.15, will reword it :)